instrumentation/grpc/src/test/java/brave/grpc/ITTracingClientInterceptor.java

```diff
@@ -50,22 +53,30 @@
   TestServer server = new TestServer();
   ManagedChannel client;
 
-  @Before public void setup() throws IOException {
+  @Before
+  public void setup() throws IOException {
     server.start();
     client = newClient();
   }
 
-  @After public void close() throws Exception {
+  @After
```

Comment:
nit square style, not google style https://github.com/square/java-code-styles

Reply:
style changes reverted